### ISSUE TITLE
Specify center coordinates for the circle and radial layouts

### ISSUE THREAD

**Issue by nklepeis** [NetLogo/NetLogo#696]
It would add flexibility in agent layouts to be able to specify the center of the layout-circle and layout-radial layouts.        This would allow for the construction of complex clusters that are centered at different places in the world.

**Comment by SethTisue** [comment]
note that as a workaround for this lack, you could just do `ask turtles [ setxy xcor + <x-offset> ycor + <y-offset>` ]`

**Comment by Idloj** [comment]
Why not providing this out-of-the-box?

**Comment by SethTisue** [comment]
I'm not sure if @fstonedahl ever considered it

**Comment by fstonedahl** [comment]
Yeah, those were implemented in the fairly early days of network support in NetLogo, and we were (or at least I was) mostly thinking about having one network globally in the model, rather than the possibility of multiple sub-networks, or hierarchical networks...  note that layout-spring also does not offer the ability to constrain the placement of the network.  In retrospect, perhaps something like specifying a bounding box for the layout should have been added for *all* the network layout primitives?  I'm not sure...

I also wonder whether more advanced/custom layout network capabilities would better be housed in an extension, such as: https://github.com/NetLogo/NW-Extension.
